include/cv/aggregator.hpp

```diff
@@ -0,0 +1,49 @@
+#ifndef INCLUDE_CV_AGGREGATOR_HPP_
+#define INCLUDE_CV_AGGREGATOR_HPP_
+
+#include "cv/utilities.hpp"
+#include "cv/pipeline.hpp"
+#include "utilities/constants.hpp"
+#include "utilities/lockptr.hpp"
+
+#include <mutex>
+#include <vector>
+#include <future>
+#include <queue>
+#include <array>
+#include <functional>
+
+struct CVResults {
+    std::vector<DetectedTarget> detected_targets;
+    // matches[0] = 5 => detected_targets[5] is matched with bottle A
+    std::array<size_t, NUM_AIRDROP_BOTTLES> matches;
```

Comment:
what is the point of using [`size_t`](https://en.cppreference.com/w/cpp/types/size_t)?

Reply:
It is more explicitly an index into a container, compared to an int which is very general.